docs/source/concepts/bento.rst

```diff
@@ -59,100 +59,105 @@ with a ``bentofile.yaml`` build file. Here's an example from the
-directory.
+For projects that are part of a larger codebase and interact with other local Python
+modules or those containing multiple Bentos/Services, it might not be possible to
+put all Service definition code and ``bentofile.yaml`` in the project's root directory.
 
-BentoML allows placing the service definition file and bentofile anywhere in the project
-directory. In this case, the user needs to provide the ``build_ctx`` and
-``bentofile`` argument to the ``bentoml build`` CLI command.
+BentoML allows the placement of the Service definition and ``bentofile.yaml`` anywhere in the project directory.
+In such scenarios, specify the ``build_ctx`` and ``bentofile`` arguments when running the ``bentoml build`` command.
 
-build_ctx
-    Build context is your Python project's working directory. This is from where you
-    start the Python interpreter during development so that your local python modules
-    can be imported properly. Default to current directory where the
-    ``bentoml build`` takes place.
+* ``build_ctx``: The build context represents the working directory of your Python project. It's the location where the Python interpreter starts,
+  ensuring the correct import of local Python modules. By default, it's set to the current directory where the ``bentoml build`` command is executed.
+* ``bentofile``: A YAML configuration file that specifies the :ref:`concepts/bento:Bento build options`. It defaults to the ``bentofile.yaml`` file in the build context.
 
-bentofile
-    ``bentofile`` is a ``.yaml`` file that specifies the
-    :ref:`concepts/bento:Bento Build Options`. Default to the ``bentofile.yaml``
-    file under the build context.
+To customize their values, use the following:
 
-They can also be customized via the CLI command, e.g.:
+.. code-block:: bash
+
+    bentoml build -f ./src/my_project_a/bento_fraud_detect.yaml ./src/
+
+Structure
+^^^^^^^^^
+
+By default, all created Bentos are stored in the BentoML Bento Store, which is essentially a local directory. You can go to a specific Bento directory by running the following command:
 
 .. code-block:: bash
 
-    » bentoml build -f ./src/my_project_a/bento_fraud_detect.yaml ./src/
+    $ cd $(bentoml get BENTO_TAG -o path)
+
+Inside the directory, you might see different files and sub-directories depending on the configurations in ``bentofile.yaml``. A typical Bento contains the following key sub-directories:
 
+* ``src``: Contains files specified in the :ref:`include <concepts/bento:Files to include>` field of ``bentofile.yaml``. These
+  files are relative to user Python code's CWD (current working directory), which makes importing relative modules and file paths inside user code possible.
+* ``apis``: Contains API definitions auto-generated from the Service's API specifications.
+* ``env``: Contains environment-related files for Bento initialization. These files are generated based on the :ref:`concepts/bento:Bento build options` specified in ``bentofile.yaml``.
```

Comment:
imo we can show the tree graph of this and have comments for specific folder fuctions.

Reply:
The reason why I did not show this is that the tree graph can be different if users are using different BentoML versions, so I only listed the key subdirectories here. For example, the `models` directory does not exist in the latest BentoML version, right?

Reply:
ah that makes sense.